builtin/stash--helper.c

```diff
@@ -694,6 +700,43 @@ static int show_stash(int argc, const char **argv, const char *prefix)
 	return ret;
 }
 
+static int store_stash(int argc, const char **argv, const char *prefix)
+{
+	struct object_id obj;
+	int ret = 0;
+	const char *stash_msg = NULL;
+	const char *w_commit = NULL;
+	struct option options[] = {
+		OPT__QUIET(&quiet, N_("be quiet, only report errors")),
+		OPT_STRING('m', "message", &stash_msg, "message", N_("stash message")),
+		OPT_END()
+	};
+
+	argc = parse_options(argc, argv, prefix, options,
+		git_stash_helper_store_usage, PARSE_OPT_KEEP_UNKNOWN);
+
+	if (argc == 0) {
```

Comment:
Should this not be `if (argc != 1)`? I mean, we are complaining that there is not exactly one `<commit>` argument, so we should not allow *two* (or more)...

Reply:
FWIW the shell version agrees:

```sh
test $# = 1 ||
die "$(eval_gettext "\"$dashless store\" requires one <commit> argument")"
```